cs/test/core/ConvertTests.cs

```diff
@@ -27,6 +27,9 @@ public class Decimals
         [global::Bond.Id(4), global::Bond.Type(typeof(global::Bond.Tag.nullable<global::Bond.Tag.blob>))]
```

Comment:
We should also ensure that we have test coverage for aliases required, optional, required_optional fields that use a type other than blob as their underlying type. Since blob is special cased in many places, we also want to test the non-special case code paths.